docs/usage.md

```diff
@@ -0,0 +1,51 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Usage
+
+The **Playbook Buttons** widget adds playbooks as separate buttons on a records detailed view that can be executed directly from the records' view panel.
+
+## Editing the Playbooks Button Widget
+
+1. Edit a record's view template and select **Add Widget** button.
+
+2. Select **Playbook Buttons** from the list to bring up the **Playbook Buttons** widget's edit view.
+
+3. Select playbooks from the **Select Action Buttons** field. You can select multiple playbooks.
+
+    ![Select playbooks dialog image](./res/edit-view-00.png)
+
+4. Assign an icon to each selected playbook.
+
+    ![Playbooks with icons added image](./res/edit-view-01.png)
+
+5. Select **Show playbook execution progress on wizard** to get real-time progress updates as playbooks run.
+
+    ![Playbook execution progress on wizard checkbox](./res/edit-view-02.png)
+
+6. Select the playbooks to execute from the **Playbook included list**.
```

Comment:
Rephrase the Sentence, this field is used to specifically for the PBs to show there execution progress on "PB execution wizard"
Also, the PBs listed here are subset of PB selected in the field "Select Action Buttons".

Reply:
Done